packages/renative-template-hello-world/src/config.js

```diff
@@ -147,7 +147,7 @@ export const themeStyles = StyleSheet.create({
 });
 
 export const ROUTES = {
-    HOME: '/',
+    HOME: 'home',
```

Comment:
This appears to be done previously to work on Web. Have you tried testing your changes with Web? It may be the case that the solution is something like `isPlatformWeb ? '/' : 'home'`.